api/main.ts

```diff
@@ -0,0 +1,183 @@
+import {
+  POLLING_INTERVAL_IN_SECONDS,
+  RANGE_BLOCK,
+  START_BLOCK,
+  WRITE_TO_CSV,
+} from '../globals';
+import {
+  instantiateCsv,
+  writeProfileToCsv,
+  writeTransactionToCsv,
+} from './csv/csv';
+import { getData } from './envio/envio';
+import { TYPE } from './interface';
+import { writeData } from './neo4j/neo4j';
+import { getLatestBlockNumber } from './utils';
+
+const main = async () => {
+  WRITE_TO_CSV ? instantiateCsv() : console.log('🌝 Writting data to NEO4J');
+
+  console.log('🏁 Starting block:', START_BLOCK);
+  const latestBlock = await getLatestBlockNumber();
+  console.log('🧱 Latest block:', latestBlock);
+
+  console.log('💁‍♀️ Starting profiles indexing');
+  await initialProcessProfiles(TYPE.PROFILES, latestBlock);
+  console.log('👌 Profiles initial sweep complete');
+
+  console.log('💸 Starting Transactions initial sweep');
+  await initialProcessTransactions(TYPE.TRANSACTIONS, latestBlock);
+  console.log('👌 Transactions initial sweep complete');
+
+  console.log(
+    `🥝 Processing new data every ${POLLING_INTERVAL_IN_SECONDS} seconds. Last synced block: ${latestBlock}`,
+  );
+
+  const lastSyncedBlock = latestBlock;
+  continuousPullingProfile(TYPE.PROFILES, lastSyncedBlock);
```

Comment:
maybe this function name should be plural? Makes more sense semantically, you constantly pull profiles, not a single one at a time.
Same comment for `continuousPullingTransactions`

```suggestion
  continuousPullingProfiles(TYPE.PROFILES, lastSyncedBlock);
```